locals.tf

```diff
@@ -18,4 +18,6 @@ locals {
       name = var.log_analytics_workspace.name
     }
   ) : null # Finally, the Log Analytics Workspace should be disabled.
-}
\ No newline at end of file
+  # If var.rbac_aad_azure_rbac_enabled  is null set the local to false. Otherise set the local to the var true/false value
+  rbac_aad_azure_rbac_enabled = var.rbac_aad_azure_rbac_enabled == null ? false : var.rbac_aad_azure_rbac_enabled
```

Comment:
Could we simplify this expression?:

```hcl
rbac_aad_azure_rbac_enabled = var.rbac_aad_azure_rbac_enabled == true
```

Reply:
Yes we can, I tested:
```
% terraform console
> null == true
false
```
So you expression works and it is simpler. Thanks